pages/feed/index.tsx

```diff
@@ -39,6 +41,12 @@ const Feeds: WithPageLayout<HighlightSSRProps> = (props: HighlightSSRProps) => {
   const { user } = useSupabaseAuth();
   const { data: repos } = useFetchHighlightRepos();
 
+  const { data: FeaturedHighlights } = useFetchFeaturedHighlights();
```

Comment:
nit: camel case variables

```suggestion
  const { data: featuredHighlights } = useFetchFeaturedHighlights();
```